var/spack/repos/builtin/packages/apfel/package.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Apfel(AutotoolsPackage):
+    """APFEL is a library able to perform DGLAP evolution up to NNLO in QCD and
+       to NLO in QED, both with pole and MSbar masses. The coupled DGLAP
+       QCD+QED evolution equations are solved in x-space by means of higher
+       order interpolations and Runge-Kutta techniques."""
+
+    homepage = "http://apfel.hepforge.org/"
```

Comment:
Why not using the URL on the GitHub page as the homepage (i.e., https://apfel.mi.infn.it/)?

(My browser does not like the URL you provided.)

Reply:
The .it URL is not a homepage, rather a live demo. I can change the URL to their github (https://github.com/scarrazza/apfel) if you prefer it that way. I have also sent an e-mail to the developers asking them to update the SSL certificate for the hepforge site.